src/core/reducers/categories.js

```diff
@@ -96,6 +104,47 @@ export function createEmptyCategoryList(): EmptyCategoryListType {
   }, {});
 }
 
+export const getCategories = (
+  categoriesState: CategoryMapType,
+  appName: string,
+  addonType: string,
+): Array<mixed> | null => {
+  if (
+    categoriesState &&
+    categoriesState[appName] &&
+    categoriesState[appName][addonType]
+  ) {
+    return Object.values(categoriesState[appName][addonType]);
+  }
+  return null;
+};
+
+export function getCategoryNames(
+  categoriesState: CategoryMapType,
+  addonCategories: AddonCategories,
```

Comment:
As mentioned above, in place of `AddonCategories`, you can use `$PropertyType<AddonType, 'categories'>`

Reply:
Yes, I will update that.